phpBB/phpbb/update/controller.php

```diff
@@ -59,8 +74,13 @@ public function __construct(
 	 */
 	public function handle(string $download): array
 	{
+		if ($this->user->data['user_type'] != USER_FOUNDER)
+		{
+			throw new http_exception(403);
```

Comment:
You might want to add a language string for this:
`throw new http_exception(403, 'INSTALL_PHPBB_NOT_INSTALLED');`

Reply:
Maybe, but it should be inaccessible for other users anyway - as in link hidden, so probably you called this thing by hand, in which case no information for you.